kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiServer.scala

```diff
@@ -42,6 +47,35 @@ object KyuubiServer extends Logging {
       zkServer.start()
       conf.set(HA_ZK_QUORUM, zkServer.getConnectString)
       conf.set(HA_ZK_AUTH_TYPE, ZooKeeperAuthTypes.NONE.toString)
+    } else {
+      // create chroot path if necessary
+      val connectionStr = conf.get(HA_ZK_QUORUM)
+      val chrootIndex = connectionStr.indexOf("/")
```

Comment:
`connectionStr` is a comma-separated list of servers to connect to.

```
localhost:2181/lakehouse,localhost:2181/lakehouse
localhost:2181/lakehouse,localhost:2181
localhost:2181,localhost:2181/lakehouse
```

Are we able to handle all these valid and invalid cases?

Reply:
> `connectionStr` is a comma-separated list of servers to connect to.
> 
> ```
> localhost:2181/lakehouse,localhost:2181/lakehouse
> localhost:2181/lakehouse,localhost:2181
> localhost:2181,localhost:2181/lakehouse
> ```
> 
> Are we able to handle all these valid and invalid cases?

In general, the third one is recommended and used, so I think we only need to support this and we can provide an example to cover this case in parameter document.

Reply:
Shall we fail the invalid cases directly？

Reply:
Sounds good, I will refine later.

Reply:
thanks~

Reply:
Done.